ci/base-tests.sh

```diff
@@ -27,6 +27,9 @@ cd clippy_dev && cargo test && cd ..
 # Perform various checks for lint registration
 ./util/dev update_lints --check
 
+echo $PATH
```

Comment:
I think this debug output can also be removed

Reply:
Removed in latest rebase